server/src/server.ts

```diff
@@ -38,6 +40,7 @@ let extensionConfiguration: extensionConfiguration = {
     enable: false,
     maxLength: 25
   },
+  codeLens: false,
```

Comment:
If we change the default to true, do we need to change the value in 2 places?
If so, a comment should go here reminding about that.

Reply:
Or is this going to be quickly overridden anyway, in which case, some comment sating it's a temporary placeholder.

Reply:
The latter; I added a clarifying comment.